CheckerApi/Utils/JobCommon.cs

```diff
@@ -0,0 +1,19 @@
+using CheckerApi.Models.Config;
+using Microsoft.Extensions.Configuration;
+using System.Net;
+
+public class JobCommon {
```

Comment:
Please move as method inside of Job. Each job inherits that class.

Reply:
done